Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -1315,11 +1311,24 @@ def fetch_incidents(client: Client, fetch_time: str, fetch_limit: str, last_run:
         ),
         limit=fetch_limit,
     )
+
+    # Some alerts have exactly the same backend_timestamp,
+    # therefor we dedup alerts that have the backend_timestamp of the last alert in the last run.
+    def get_backend_timestamp(alert):
+        return alert['backend_timestamp']
+    alert_ids_grouped_by_backend_timestamp = map_reduce(response['results'], get_backend_timestamp)
+    # backend_timestamp of last alert retrieved.
+    last_backend_timestamp = response['results'][-1]['backend_timestamp']
```

Comment:
I don't like the "backend" in the variable name, maybe change to "last_fetched_alert_timestamp".